src/parser/parser.py

```diff
@@ -88,7 +88,8 @@ def _merge_localizations(self, language, group, data):
             # duplicate key error. This is a temporary measure to keep patch updates going
             elif group != 'heroes':
                 current_value = self.localizations[language][key]
-                raise Exception(
```

Comment:
Should this be changed? Genuinely not sure what to do about it

Reply:
Yeah, when I have it replace it print it out correctly, I gave a clean try at is here: #66

Reply:
Now I'm not sure..... I'll revert it, it's probably a linux thing